Description: Write text to .write() rather than .writelines()
Author: Jelmer Vernooij <jelmer@debian.org>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=722091
Status: not (yet) forwarded upstream

=== modified file 'bzrlib/doc_generate/autodoc_rstx.py'
--- a/bzrlib/doc_generate/autodoc_rstx.py	2011-12-19 13:23:58 +0000
+++ b/bzrlib/doc_generate/autodoc_rstx.py	2014-04-27 21:54:41 +0000
@@ -131,7 +131,7 @@
     topic_id = "%s-%s" % (topic, "help")
     filename = bzrlib.osutils.pathjoin(output_dir, topic_id + ".txt")
     f =  open(filename, "w")
-    f.writelines(text)
+    f.write(text)
     f.close()
     return topic_id
 

